package.json

```diff
@@ -1,13 +1,16 @@
 {
-  "name": "integration-nameOfYourIntegration",
-  "version": "1.0.0",
+  "name": "integration-GoogleTagManager",
+  "version": "2.0.0",
+  "repository": "https://github.com/mparticle-integrations/mparticle-javascript-integration-google-tag-manager",
+  "browser": "build/GoogleTagManager-Kit.js",
```

Comment:
change this to dist/GoogleTagManager-Kit.js

and add a "files": ['dist/GoogleTagManager-Kit.js'] as well.  take a look at as https://github.com/mparticle-integrations/mparticle-javascript-integration-amplitude/blob/npm-module/package.json an example